sycl/test/warnings/sycl_2020_deprecations.cpp

```diff
@@ -194,6 +194,17 @@ int main() {
   // expected-warning@+1{{'get_linear_id' is deprecated: use sycl::group::get_group_linear_id() instead}}
   group.get_linear_id();
 
+  // expected-warning@+1{{'default_selector' is deprecated: Use the callable sycl::default_selector_v instead.}}
+  sycl::default_selector ds{};
```

Comment:
```suggestion
  sycl::default_selector ds;
```
can be simplified everywhere.